docs/adr/0007-alerting-setup.md

```diff
@@ -0,0 +1,49 @@
+
+Alerting with Prometheus is separated into two parts. Alerting rules in Prometheus servers send alerts to an [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). The Alertmanager then manages those alerts, including silencing, inhibition, aggregation and sending out notifications via methods such as email, on-call notification systems, and chat platforms.
+
+There are multiple on-call notification systems such as [PagerDuty](https://www.pagerduty.com/), [JIRA](https://www.atlassian.com/software/jira) etc that can be used for incident reporting, but which of these tools are best suitable for reporting the outages/incidents to our users?
+
+## Decision Drivers <!-- optional -->
+
+* Visibility of incident reporting
+  * How can the incidents be tracked and reported in an open and transparent manner for our users?
+* Compatibility with Prometheus
+  * Is the incident reporting tool compatible with Prometheus i.e. can it handle/receive Prometheus alerts?
+* Complexity and cost of incident reporting tool
+  * How easy/hard is it to manage and operate the incident reporting tool?
+  * Is it a free or paid tool?
+
+## Considered Options
+
+* Option 1:
+  * Use [PagerDuty](https://www.pagerduty.com/) which is a popular paid on-call management and incident response platform
+* Option 2:
+  * Use open source tools like:
+    * [Cabot](https://github.com/arachnys/cabot) - Python/Djano based monitoring platform
+    * [OpenDuty](https://github.com/openduty/openduty) - Incident escalation tool similar to PagerDuty
+    * [Dispatch](https://github.com/Netflix/dispatch) - Incident management tool by Netflix
+    * [Response](https://github.com/monzo/response) - Django based incident management tool
+
+  which are free, self-hosted infrastructure that provides some of the best features of PagerDuty, Pingdom etc without their cost and complexity
+* Option 3:
+  * Use [GitHub Alertmanager receiver](https://github.com/m-lab/alertmanager-github-receiver) which is a Prometheus Alertmanager webhook receiver that creates GitHub issues from alerts
+
+## Decision Outcome
+Chosen option: **Option 3**, because:
+  * The [GitHub alertmanager receiver](https://github.com/m-lab/alertmanager-github-receiver) can easily be configured and operated to function with Prometheus alerts. It automatically creates issues in GitHub repositories for any active alerts being fired, making it visible for any user to track
+  * All communication/updates/concerns related to the incident can be easily handled by adding comments in the issues created by the GitHub receiver
+  * Unlike Option 1, there is no additional cost involved
+  * There is no requirement for using JIRA/Slack for incident tracking, which are the only supported options in some of the tools listed in Option 2 (such as [Dispatch](https://github.com/Netflix/dispatch) and [Response](https://github.com/monzo/response))
```

Comment:
Piggybacking on this: And in case that such requirement surfaces, we can use Github bots for different platforms like [Github for Slack](https://slack.github.com/), [Google Chat](https://support.google.com/chat/answer/9632291?co=GENIE.Platform%3DAndroid&hl=en) etc. so we're notified on the issues asap.

Reply:
updated :)